docs/Makefile

```diff
@@ -19,8 +20,22 @@ all: dirhtml
 # Setup commands
 .PHONY: setupenv
 setupenv:
-	pip install -q poetry
 	sudo apt-get install gcc python3-dev libev4 libev-dev
+	@while IFS= read -r line; do \
+      if [[ "$${line}" =~ ^[[:space:]]*requires[[:space:]]*= ]]; then\
+        content="$${line#*=}";\
+        content="$${content//[[:space:]]/}";\
+        content="$${content#[}";\
+        content="$${content%]}";\
+        IFS=',' read -ra items <<< "$$content";\
+        for item in "$${items[@]}"; do\
+          item="$${item%\"}";\
+          item="$${item#\"}";\
+          pip install "$$item";\
+        done;\
+      fi;\
+    done < pyproject.toml
+
```

Comment:
Oh wow this is such an ugly hack. You parse pyproject.toml, and manually install deps specified there.
I really don't think this is a good idea - managing dependencies is what all tools like poetry / uv are for. Let's not write our own and make the build system / CI etc more cursed than it needs to be.

Reply:
actually, since uv is already adobpted it could be a good option, let me try

Reply:
please check